tests/test_empty_content.py

```
@@ -0,0 +1,118 @@
+    print("  OpenAI: OK (None -> ERROR, Text -> RECEIVED)")
+
+
+def test_anthropic():
+    try:
+        import anthropic  # noqa: F401
+    except ImportError:
+        print("  Anthropic: SDK nicht installiert -> uebersprungen")
+        return
+    from src.core.anthropic_client import AnthropicClient
+    def block(text):
+        b = MagicMock(); b.text = text; return b
```

Ruff (0.15.14)

[error] Multiple statements on one line (semicolon)

(E702)